docs/src/frontend/index.md

```diff
@@ -1123,6 +1123,16 @@ stdlib.parseInt(Int) => number
 
 Parses a signed `{!rsh} Int` into a JavaScript number.
 
+Example:
+
+```js
+load: /examples/programmer-art/src/index.mjs
```

Comment:
@EBEREGIT I've grepped around the source code and found an obscure example at the end of https://github.com/reach-sh/reach-lang/blob/master/rpc-client/js/index.mjs

It may be best to create your own `parseInt` sample and place it in a new directory in `examples/`

If you need help, @hamirmahal should be able to lend a hand.

Reply:
#1115 offers an alternative.